Steps/UploadCodeCoverage.yml

```diff
@@ -145,14 +152,20 @@ steps:
         cmd = f'{UPLOAD_CMD} -f {cov_file} -Z'
         if COV_FLAG:
           cmd += f' -F {COV_FLAG}'
-        process = subprocess.Popen(
-          cmd,
-          stdout=subprocess.PIPE,
-          stderr=subprocess.PIPE,
-          shell=True)
-        output, error = process.communicate()
-        print(f"##[debug]{output.decode('utf-8')}")
+        loop = 0
+        process = 0
+        while loop < 5 and process == 0:
+          process = subprocess.Popen(
```

Comment:
we probably want to add a delay, similar to the `Retry()` code you added above. The `Retry()` code doubles the delay time between retries each iteration due to the backoff_factor of 1.

Reply:
Added code and used time.sleep. 

Matched documented method of [retry](https://urllib3.readthedocs.io/en/latest/reference/urllib3.util.html#module-urllib3.util.retry)